stingray/tests/test_crossspectrum.py

```diff
@@ -262,6 +262,15 @@ def test_rebin_error(self):
         with pytest.raises(ValueError):
             cs.rebin()
 
+    def test_fullspec(self):
+        csT = Crossspectrum(self.lc1, self.lc2, fullspec=True)
+        assert csT.fullspec == True
+        assert self.cs.fullspec == False
+        assert csT.n == self.cs.n
+        assert csT.n == len(csT.power)
+        assert self.cs.n != len(self.cs.power)
+        assert len(csT.power) >= len(self.cs.power)
```

Comment:
can you also check that the length of power is the same as the length of the lightcurves (for fullspec=True) please!